reconcile/cli.py

```diff
@@ -1546,6 +1546,33 @@ def ldap_users(ctx, infra_project_id, app_interface_project_id):
     )
 
 
+@integration.command(short_help="Manages raw HCL Terraform from a separate repository.")
+@click.option(
+    "-d",
+    "--output-dir",
+    help="Specify a directory to individually output each repo plan to for the executor",
+)
+@click.option(
```

Comment:
Curious when would we not validate git?

Reply:
Good point. I added that option to avoid calling the check_ref function https://github.com/app-sre/qontract-reconcile/pull/3464/files#diff-05e8f500c44ae2dc8a3b8087020cc1d2946506df84ea9a118e110c8d8ad82ba2R169-R180 during testing, although I don't think that's the correct pattern and I should probably just mock the GitLab API and query calls. If the integration can't reach and validate Git, then the executor also wouldn't be able to

Reply:
You could keep the parameter input but not expose it to the CLI?

Reply:
That works too